pages/faq.js

```diff
@@ -142,7 +139,7 @@ export default function Faq({ hackathons }) {
         id: faqID,
       },
     });
-    setNewFaq({});
+    setNewFaq({ hackathonIDs: [] });
```

Comment:
maybe `selectedHackathonIDs`? Also maybe this could be a map for faster lookup? Then we don't have to use `.includes` so many times?

Reply:
I just wanted to keep the variable names consistent with the firebase fields so I kept it as `hackathonIDs`
![Screen Shot 2020-10-16 at 11 08 01 PM](https://user-images.githubusercontent.com/38872354/96329710-7263f880-1004-11eb-94a0-8d234fe37f29.jpg)
I think it'd be easier to keep it as an array since it's the data type in the db. When we fetch a faq, we'll need to convert the array into an object, then when we pull up a modal, we need to fill in the object with hackathons are weren't selected and set them to false. Then when we write to the db we need to filter out all of the hackathons that weren't selected. I think it's a little bit more processing to convert between data types? Because we only have a few hackathons I think running includes is a reasonable tradeoff here. But I get that map/object is more robust/standard. Do you think this is a reasonable tradeoff?

Reply:
sounds good!